modules/sandboxed-containers-deleting-operator-deployment-cr.adoc

```diff
@@ -0,0 +1,30 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-operator-deployment-cr_{context}"]
+= Deleting the {sandboxed-containers-operator} deployment custom resource
+
+To uninstall {sandboxed-containers-first}, you must first delete the {sandboxed-containers-first} custom resource `KataConfig`. This removes and uninstalls the `kata` runtime and its related resources from your cluster.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. From the {product-title} web console, select `openshift-sandboxed-containers` from the *Projects* list.
+. Navigate to the *Operators* → *Installed Operators* page.
+. Click *{sandboxed-containers-first}*.
+. Click the *{sandboxed-containers-operator}* tab.
+. Click the scroll-down list in the *Operator Details*, and click *Delete `KataConfig`*.
+. Click *Delete* in the confirmation window.
+. Optional. You can use the OpenShift CLI (`oc`) to retrieve all pods that use `kata` as `runtimeClassName` by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -o json | jq -r '.items[] | select(.spec.runtimeClassName | test("kata")).metadata.name'
+----
++
+You can delete all `pod`, `deployment`, `DeploymentConfig`, `ReplicaSet`, `ReplicationController`, and `StatefulSet` objects that use `kata` as `runtimeClass`.
```

Comment:
This should be a continuation of the previous step. I think we could remove step number 7 and what follows, as it might be more problematic to have it in, also I tested this command locally and it did not work for some reason.

Reply:
@zanetworker - I would have to have it as a "Optional" step as in continued with step number 7. I'm open to suggestions though. This is what was suggested by my colleagues when I asked them. :-)